Examples/Algorithms/TrackFitting/src/GsfFitterFunction.cpp

```diff
@@ -111,38 +111,25 @@ std::shared_ptr<TrackFittingAlgorithm::TrackFitterFunction>
 ActsExamples::makeGsfFitterFunction(
     std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry,
     std::shared_ptr<const Acts::MagneticFieldProvider> magneticField,
-    std::string lowBetheHeitlerPath, std::string highBetheHeitlerPath,
+    Acts::Experimental::AtlasBetheHeitlerApprox<6, 5> betheHeitlerApprox,
     std::size_t maxComponents, bool abortOnError,
     bool disableAllMaterialHandling) {
   MultiStepper stepper(std::move(magneticField));
 
-  const auto bhapp = [&]() {
-    if (lowBetheHeitlerPath.empty() && highBetheHeitlerPath.empty()) {
-      return Acts::Experimental::makeDefaultBetheHeitlerApprox();
-    } else if (std::filesystem::exists(lowBetheHeitlerPath) &&
-               std::filesystem::exists(highBetheHeitlerPath)) {
-      return Acts::Experimental::AtlasBetheHeitlerApprox<6, 5>::loadFromFile(
-          lowBetheHeitlerPath, highBetheHeitlerPath);
-    } else {
-      throw std::invalid_argument(
-          "Paths to bethe heitler parameterization do not exist. Pass empty "
-          "strings to load a default parameterization");
-    }
-  }();
-
   // Standard fitter
   Acts::Navigator::Config cfg{trackingGeometry};
   cfg.resolvePassive = false;
   cfg.resolveMaterial = true;
   cfg.resolveSensitive = true;
   Acts::Navigator navigator(cfg);
   Propagator propagator(std::move(stepper), std::move(navigator));
-  Fitter trackFitter(std::move(propagator), BHApprox(bhapp));
+  Fitter trackFitter(std::move(propagator), BHApprox(betheHeitlerApprox));
```

Comment:
should we use `move` even tho it might be a copy? or make the copy implicit?

```suggestion
  Fitter trackFitter(std::move(propagator), std::move(betheHeitlerApprox));
```

Reply:
Hmm the problem with the move is that we want to instaniate two fitters. If we do `std::move`, it might be that the second fitter gets an invalid object

Reply:
I somehow want to make the copy explicit, if its ok for you